Orange/widgets/unsupervised/owhierarchicalclustering.py

```diff
@@ -1044,7 +1045,7 @@ def send_report(self):
         elif self.selection_method == 1:
             sel = "at {:.1f} of height".format(self.cut_ratio)
         else:
-            sel = f"top {self.top_n} clusters"
+            sel = f"top {self.top_n} {pl(self.top_n, 'cluster')}"
```

Comment:
This change make sense for English but, by the way, in case you didn't know: even if it remained as it was, could still use `plsi` in Slovenian translation.

Even if the string is initially not an f-string (this one is), trubar changes it to f-string, if necessary.